### PR TITLE
docs(website): Update troubleshooting procedures in node-troubleshooting.mdx

### PR DESCRIPTION
**Pull request by rakkooo**

### CODE REVIEW

**Comment by TIGGERLAND** [comment]
While both errors can be resolved by executing 'cd simple-taiko-node', they present different error messages.

File: packages/website/pages/docs/reference/node-troubleshooting.mdx [hunk 1]
```diff
@@ -74,7 +74,11 @@ This seems to be a common issue if you're using Alchemy endpoints, switching to
 
 #### `no configuration file provided: not found`
 
-You are in the wrong directory, `cd simple-taiko-node` and run the command again.
+This command is searching for the docker-compose.yml file.You are in the wrong directory, `cd simple-taiko-node` and run the command again.
```
Comment (d1onys1us) [inline packages/website/pages/docs/reference/node-troubleshooting.mdx:77]:
```suggestion
This command is searching for the docker-compose.yml file. You are in the wrong directory, `cd simple-taiko-node` and run the command again.
```

File: packages/website/pages/docs/reference/node-troubleshooting.mdx [hunk 2]
```diff
@@ -74,7 +74,11 @@ This seems to be a common issue if you're using Alchemy endpoints, switching to
...
+
+#### `Couldn't find env file`
+
+This command is searching for the .env file.You are in the wrong directory, `cd simple-taiko-node` and run the command again.
```
Comment (d1onys1us) [inline packages/website/pages/docs/reference/node-troubleshooting.mdx:81]:
```suggestion
This command is searching for the .env file. You are in the wrong directory, `cd simple-taiko-node` and run the command again.
```

**Review by dionysuzx — APPROVED** [review]
two small

**Comment by TIGGERLAND** [comment]
Thanks for approving.

**Review by MarcusWentz — APPROVED** [review]

**Pull request merged**